### PR TITLE
README: typo and grammer fixed

### PR DESCRIPTION
**Pull request by MoodyArco**
As title, the typo and grammer are checked and polished.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+It is a research project in the framework of Wadden Sea Archive Project (WASA). We collect the XRF profiles and core descriptions of 92 sediment cores from tidal flats, channels, and off-shore around the island of Norderney in order to develop an automatic facies recognition model. It is also an approach to utilize machine learning and data analysis methods (SciPy ecosystem in Python) on the geosciences data. The works are conducted by jupyter notebook and python script. The jupyter notebooks are mainly used for analyzing data and some trials, while the python scripts are the heavy works submitted to the high performance computation in University of Bremen. 
 
 ## Why open this project on Github
 I'm a PhD student started this development from spring 2019. Our research group is GEOPOLAR, University of Bremen. During this period, some results are achieved. I think it's time to share my workflow and scripts with the open source community because I've been benifited a lot from the community.
```
Comment (dispink) [inline README.md]: Please help me modify the sentence, 

> Our research group is GEOPOLAR, University of Bremen. 

to
> I have GEOPOLAR (University of Bremen) and ENL (National Taiwan University) to jointly support my study.

**Review by dispink — COMMENTED** [review]
Just a minor comment and it's my practice of giving reviews.

File: README.md (hunk 1 above)
Comment (MoodyArco) [reply README.md]: I've done the work and it seems like the request doesn't need to be re-request. 
Please check if there are any changes.

**Review by dispink — APPROVED** [review]
LGTM!

**Pull request merged**